### PR TITLE
New upstream release v1.3.1

### PR DESCRIPTION
**Pull request by runlevel5**
## Changes

* Bump to version 1.3.1
* Re-enable ppc64le

### CODE REVIEW

**Comment by runlevel5** [comment]
cc @leigh123linux @sergiomb2

File: vcmi.spec [hunk 1]
```diff
@@ -88,7 +88,7 @@ sed -i 's/GITDIR-NOTFOUND/%{version}/' cmake_modules/*
   -DCMAKE_INSTALL_RPATH_USE_LINK_PATH=ON \
   -DCMAKE_INSTALL_RPATH=%{_libdir}/%{name}
 
-%ifnarch %{ix86} x86_64 aarch64
+%ifnarch %{ix86} x86_64 aarch64 ppc64le
```
Comment (leigh123linux) [inline vcmi.spec]: Why add ppc64le to parallel builds, the builders haven't got the RAM to do it.

Comment (runlevel5) [reply vcmi.spec]: @leigh123linux Sorry I am not aware of that. Let me remove that pp64le from matrix

**Comment by runlevel5** [comment]
@leigh123linux I've addressed your feedback.

**Comment by leigh123linux** [comment]
Merged https://github.com/rpmfusion/vcmi/commit/3c00e00cc9675135080ee716f81b211c145030ed

**Comment by runlevel5** [comment]
Great thanks @leigh123linux

**Comment by kwizart** [comment]
@runlevel5 Would you be interested in maintaining this package directly yourself ?
It's currently orphaned in our database: https://admin.rpmfusion.org/pkgdb/package/free/vcmi/

See https://rpmfusion.org/Contributors

**Comment by runlevel5** [comment]
@kwizart sure I can take that on. I will look into how to join

**Pull request closed without merging**